src/views/CTLViewsHoC.jsx

```diff
@@ -82,6 +80,9 @@ const CTLViewsHoC = (WrappedView) => {
 					navigate(
 						`${CodeOrPurlPath({ codeOrPurl: prettyUrlName })}${queryString}`
 					);
+				} else {
+					setHasBeenRedirected(false);
+					navigate(`${CodeOrPurlPath({ codeOrPurl: prettyUrlName })}${search}`);
```

Comment:
I don't think we need this whole else statement and its contents. We should only be setting hasBeenRedirected and redirecting if the paramType === code, which is covered by the if(). Otherwise, we don't need to do anything.

Reply:
Ok will do